wandering-warriors/main.py

```diff
@@ -28,8 +35,70 @@ class OperationsBar(Widget):
     pass
 
 
-class CuneiformDrawingInput(Widget):
-    pass
+def calculate_points(x1, y1, x2, y2, steps=1):
+    dx = x2 - x1
+    dy = y2 - y1
+    dist = sqrt(dx * dx + dy * dy)
```

Comment:
`dx * dx + dy * dy`... WHY? Please use `dx ** 2 + dy ** 2`!

Also, please don't intentionally shorten variable names. That makes them harder to keep track of.